src/gtl/components/data-table/data-table.html

```diff
@@ -0,0 +1,95 @@
+<div>
+  <div class="dataTables_header miq-data-tables-header" ng-if="tableCtrl.rows.length > 0">
+    <div class="row">
+      <div class="pull-right">
+        <div>
+          <label>{{tableCtrl.perPage.label}}: </label>
+          <miq-toolbar-list on-item-click="tableCtrl.perPageClick(item)"
+                            toolbar-list="tableCtrl.perPage"></miq-toolbar-list>
+        </div>
+        <div>
+          Some text sorted by
+        </div>
+      </div>
+    </div>
+  </div>
+  <table class="table table-bordered table-striped table-hover mig-table-with-footer mig-table">
+    <thead>
+    <tr>
+      <th class="narrow miq-select">
+        <input ng-if="tableCtrl.rows.length !== 0" type="checkbox" ng-model="isChecked" ng-click="tableCtrl.onCheckAll(isChecked)" title="Select all" />
+      </th>
+      <th ng-if="$index !== 0"
+          ng-repeat="column in tableCtrl.columns track by $index"
+          ng-click="tableCtrl.onSortClick($index, !!tableCtrl.settings.sortBy && !tableCtrl.settings.sortBy.isAscending)"
+          ng-class="tableCtrl.getColumnClass(column)">
+        {{column.text}}
+        <div class="pull-right" ng-if="tableCtrl.isFilteredBy(column)" >
+          <i class="fa" ng-class="tableCtrl.getSortClass()"></i>
+        </div>
+      </th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr ng-repeat="row in tableCtrl.rows"
+        ng-class="{active : row.selected}"
+        ng-click="vm.onRowClick({$event: $event, rowData: row})">
+      <td ng-repeat="(columnKey, column) in tableCtrl.columns" ng-class="{narrow: row.cells[columnKey].is_checkbox}">
+        <input ng-if="row.cells[columnKey].is_checkbox"
+               ng-click="tableCtrl.onRowSelected($event, isSelected, row)"
+               onclick="event.stopPropagation();"
+               type="checkbox"
+               ng-model="isSelected"
+               name="check_{{row.id}}"
+               value="{{row.id}}"
+               ng-checked="row.checked"
+               class="list-grid-checkbox">
+        <i ng-if="row.cells[columnKey].icon && tableCtrl.isIconOrImage(row, columnKey)"
+           class="{{row.cells[columnKey].icon}}"
+           title="row.cells[columnKey].title"></i>
+        <img ng-if="row.cells[columnKey].icon === null && tableCtrl.isIconOrImage(row, columnKey)"
+             ng-src="{{row.img_url}}"
+             alt="{{row.cells[columnKey].title}}"
+             title="{{row.cells[columnKey].title}}" />
+        <span ng-if="row.cells[columnKey].text">
+              {{row.cells[columnKey].text}}
+          </span>
+      </td>
+    </tr>
+    </tbody>
+  </table>
+  <div class="dataTables_footer">
+    <div class="dataTables_paginate paging_bootstrap_input">
+      <ul class="pagination">
+        <li ng-class="{disabled: tableCtrl.currentPageView === 1}"
+            class="first"
+            ng-click="tableCtrl.setPage(1)">
+          <span class="i fa fa-angle-double-left"></span>
```

Comment:
Yeah, I copied this from old implementation of table with paging, that's why it's there. But I'll remove it (not really necessary there).